dev-docs/docs/@excalidraw/excalidraw/api/children-components/main-menu.mdx

```diff
@@ -167,3 +167,35 @@ function App() {
 | Prop | Type | Required | Default | Description |
 | --- | --- | :-: | :-: | --- |
 | `children ` | `React.ReactNode` | Yes | - | The content of the `Menu Group` |
+
+### MainMenu.Sub
```

Comment:
@barnabasmolnar the docs should go into a separate PR as merging this would make the docs live without shipping the package version.
Hence for docs you need to creating a PR against the branch [docs-next](https://github.com/excalidraw/excalidraw/tree/docs-next) and once all docs changes are merged to `docs-next` branch we will merge the branch `docs-next` to master when we ship the version.
I will document this in docs as well for better visibility

Reply:
Noted, thanks!